epub33/rs/index.html

```diff
@@ -1866,6 +1866,9 @@ <h5>Rendering EPUB content document elements</h5>
 			<section id="sec-behaviors-interaction">
 				<h4>Interacting with the EPUB content document</h4>
 
+				<p class="note">Earlier versions of this specification included some information about embedded audio and 
+					video [epubmediaoverlays-32]. This feature has been deprecated.</p>
```

Comment:
```suggestion
					video [[epubmediaoverlays-32]]. This feature has been deprecated.</p>
```

and a reference to epubmediaoverlays-32 must be added to the bibliography part on the respec structure of the header.

(The `[[...]]` forces respec to use the bibliography entry in the references.)